plugins/sentinelone/help.md

```diff
@@ -1054,6 +1050,8 @@ Example input:
 
 ```
 {
+  "blacklist_state": true,
```

Comment:
Have you changed "Blacklist by Content Hash" to have the same inputs as "Blacklist"?
Or is this a mistake?
Blacklist by Content Hash has only hash as an input in 7.0.0

Reply:
Yes, it's a mistake. I changed it already